icevision/metrics/confusion_matrix/confusion_matrix.py

```diff
@@ -0,0 +1,153 @@
+__all__ = ["SimpleConfusionMatrix"]
+
+from icevision.data.prediction import Prediction
+from icevision.metrics.metric import Metric
+from icevision.imports import *
+from icevision.metrics.confusion_matrix.confusion_matrix_utils import *
+from pytorch_lightning import loggers as pl_loggers
```

Comment:
this import is dangerous, if lightning is not installed this will thrown an error when doing `from icevision.all import *`

Reply:
so, I think we can delete this line, `pl` will already come from `imports`